README.md

```diff
@@ -154,7 +155,7 @@ Example command:
 
 ### Run tests
 
-To run tests, run the following command:  npx stylelint "**/*.{css,scss}".
+To run tests, run the following command:  npx stylelint "**/*.{css,scss}" and npx eslint .
```

Comment:
- [ ] [optional] @AjroudiRida kindly consider adding @hetrox8 as one of your authors , this makes the user to see all individuals who have contributed to this repo.